Python/Setup/swix/Packages/Microsoft.PythonTools.Profiling.Vsix.swixproj

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I have, and it is (after a few attempts...). Managed a successful install earlier today :) But this is the part I was referring to last week when I mentioned the extra level of indirection.

---
In reply to: [69643900](https://github.com/Microsoft/PTVS/pull/1396#discussion_r69643900) [](ancestors = 69643900)